docs/helm-chart/production-guide.rst

```diff
@@ -200,3 +200,24 @@ By default, the chart will deploy Redis. However, you can use any supported Cele
 
 For more information about setting up a Celery broker, refer to the
 exhaustive `Celery documentation on the topic <http://docs.celeryproject.org/en/latest/getting-started/>`_.
+
+Security Context Constraints
+-----------------------------
+
+A ``Security Context Constraint`` (SCC) is a OpenShift construct that works as a RBAC rule however it targets PODs instead of users.
+When defining a SCC, one can control actions and resources a POD can perform or access during startup and runtime.
```

Comment:
```suggestion
A ``Security Context Constraint`` (SCC) is a OpenShift construct that works as a RBAC rule however it targets Pods instead of users.
When defining a SCC, one can control actions and resources a Pod can perform or access during startup and runtime.
```

Reply:
Changes in place.